### PR TITLE
Feature/add unit test

### PR DESCRIPTION
**Pull request by Aias00**
Fixes #ISSUSE_ID.

Changes proposed in this pull request:
  -

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [x] I have made corresponding changes to the documentation.
- [x] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by Aias00** [comment]
Hi @taojintianxia , please review and let me know if any changes are required in this PR. Thanks!

File: kernel/metadata/core/src/test/java/org/apache/shardingsphere/metadata/persist/MetaDataPersistServiceTest.java [hunk 1]
```diff
@@ -66,4 +84,18 @@ private void setField(final String name, final Object value) throws ReflectiveOp
     void assertLoadDataSourceConfigurations() {
         assertTrue(metaDataPersistService.loadDataSourceConfigurations("foo_db").isEmpty());
     }
+    
+    @Test
+    void testPersistGlobalRuleConfiguration() {
```
Comment (zhaojinchao95) [inline kernel/metadata/core/src/test/java/org/apache/shardingsphere/metadata/persist/MetaDataPersistServiceTest.java:89]: The unit test not assert

File: kernel/metadata/core/src/test/java/org/apache/shardingsphere/metadata/persist/MetaDataPersistServiceTest.java [hunk 2]
```diff
@@ -66,4 +84,18 @@ private void setField(final String name, final Object value) throws ReflectiveOp
...
+        doNothing().when(globalRuleService).persist(anyCollection());
+        doNothing().when(propsService).persist(any());
+        metaDataPersistService.persistGlobalRuleConfiguration(Collections.emptyList(), new Properties());
+    }
+    
+    @Test
+    void testPersistConfigurations() {
```
Comment (zhaojinchao95) [inline kernel/metadata/core/src/test/java/org/apache/shardingsphere/metadata/persist/MetaDataPersistServiceTest.java:96]: The unit test not assert

File: kernel/metadata/core/src/test/java/org/apache/shardingsphere/metadata/persist/service/config/database/datasource/DataSourceNodePersistServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    
+    @Test
+    void testLoad() {
+        Map<String, DataSourcePoolProperties> map = dataSourceNodeService.load("123", "234");
+        Assertions.assertTrue(map.isEmpty());
+    }
+    
+    @Test
+    void testDelete() {
+        dataSourceNodeService.delete("123", "234");
```
Comment (zhaojinchao95) [inline kernel/metadata/core/src/test/java/org/apache/shardingsphere/metadata/persist/service/config/database/datasource/DataSourceNodePersistServiceTest.java:58]: The unit test not assert

File: kernel/metadata/core/src/test/java/org/apache/shardingsphere/metadata/persist/service/config/database/datasource/DataSourceNodePersistServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    private DataSourceNodePersistService dataSourceNodeService;
+    
+    @BeforeEach
+    void setUp() throws ReflectiveOperationException {
+        dataSourceNodeService = new DataSourceNodePersistService(mock(PersistRepository.class));
+    }
+    
+    @Test
+    void testPersist() {
+        dataSourceNodeService.persist("123", Collections.emptyMap());
```
Comment (zhaojinchao95) [inline kernel/metadata/core/src/test/java/org/apache/shardingsphere/metadata/persist/service/config/database/datasource/DataSourceNodePersistServiceTest.java:47]: The unit test not assert

**Comment by zhaojinchao95** [comment]
@Aias00 Hi, thank you for you pull request, this pr has some problem, you can refer to [doc](https://shardingsphere.apache.org/community/en/involved/conduct/code/)

**Comment by Aias00** [comment]
> @Aias00 Hi, thank you for you pull request, this pr has some problem, you can refer to [doc](https://shardingsphere.apache.org/community/en/involved/conduct/code/)

@zhaojinchao95 thx for the remind, i will fix them properly later

**Comment by Aias00** [comment]
> @Aias00 Hi, thank you for you pull request, this pr has some problem, you can refer to [doc](https://shardingsphere.apache.org/community/en/involved/conduct/code/)

@zhaojinchao95 Hi, i have redone the codes , pls help me review the again, thx 😊

**Pull request closed without merging**